themes/theme-boilerplate/src/scss/pages/_profile.scss

```diff
@@ -449,7 +449,10 @@ body.Section-EditProfile {
     }
 
     .Preferences > form > div {
-        overflow-x: scroll;
+
+        @include maxWidth {
```

Comment:
This doesn't work, see screen cap.

<img width="1031" alt="Screen Shot 2019-12-04 at 15 40 56" src="https://user-images.githubusercontent.com/19651440/70179744-9ba9bd80-16ac-11ea-8ca8-d3232aa574c0.png">

I really don't think it's worth adding the media query here. it should be:

```
    .Preferences > form > div {
        overflow-x: auto;
	}
```

First of all, your break point is much too wide to target only mobile devices and second of all, you won't see the scroll bar on mobile, because it'll be under the fold.